zcash_client_backend/Cargo.toml

```diff
@@ -75,10 +75,14 @@ shardtree.workspace = true
 
 # - Test dependencies
 proptest = { workspace = true, optional = true }
+jubjub = { workspace = true, optional = true }
 
 # - ZIP 321
 nom = "7"
 
+# - Runtime
+tokio.workspace = true
```

Comment:
Add two separate entries for this in `[dev-dependencies]` that enable either `rt` or `rt-multi-thread` depending on whether the build target is WASM or not.

Reply:
see comment above